rdl/data_sources/SourceTableInfo.py

```diff
@@ -0,0 +1,7 @@
+from rdl.data_sources.ChangeTrackingInfo import ChangeTrackingInfo
+
+
+class SourceTableInfo:
```

Comment:
this type doesn't belong in the `data_sources` folder.. but same for `ChangeTrackingInfo`.. so landed this new type at the same place to avoid changing more than required for our MVP

Reply:
agree it shouldnt be here.. its a bit of a mess